Cython/Includes/cpython/datetime.pxd

```diff
@@ -161,6 +172,27 @@ cdef inline object datetime_new(int year, int month, int day, int hour, int minu
 cdef inline object timedelta_new(int days, int seconds, int useconds):
     return PyDateTimeAPI.Delta_FromDelta(days, seconds, useconds, 1, PyDateTimeAPI.DeltaType)
 
+# Create a timezone from an offset, optionally a name.
+cdef inline object timezone_new(offset, name):
+    return PyDateTimeAPI.TimeZone_FromTimeZone(offset, name)
```

Comment:
"optionally" suggests to me that it should be … optional. :)
```suggestion
# Create a timezone from an offset, optionally a name.
cdef inline object timezone_from_offset(offset, name=None):
    return PyDateTimeAPI.TimeZone_FromTimeZone(offset, NULL if name is None else <PyObject*>name)
```

Reply:
I was originally thinking that the optionality would end up in the C Api's hands, which does have a default. Good catch.